cloudify_cli/commands/replace_certificates.py

```diff
@@ -0,0 +1,251 @@
+########
+# Copyright (c) 2020 Cloudify.co Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+############
+
+import os
+
+import yaml
+
+from .. import env
+from ..cli import cfy
+from ..utils import get_dict_from_yaml
+from ..exceptions import CloudifyCliError
+from ..replace_certificates_config import (raise_errors_list,
+                                           ReplaceCertificatesConfig)
+
+CERTS_CONFIG_PATH = 'certificates_replacement_config.yaml'
+
+
+@cfy.group(name='replace-certificates')
+@cfy.options.common_options
+def replace_certificates():
+    """
+    Handle the certificates replacement procedure
+    """
+    if not env.is_initialized():
+        env.raise_uninitialized()
+
+
+@replace_certificates.command(name='generate-config',
+                              short_help='Generate the configuration file '
+                                         'needed for certificates replacement')
+@cfy.options.output_path
+@cfy.assert_manager_active()
+@cfy.pass_client()
+@cfy.pass_logger
+def get_replace_certificates_config_file(output_path,
+                                         logger,
+                                         client):
+    # TODO: Take care of the AIO case
+    output_path = output_path if output_path else CERTS_CONFIG_PATH
```

Comment:
Is there a reason this isn't a default argument?

Reply:
Well, only if the user wants to save it to somewhere else

Reply:
Right, but rather than putting:
def func(something):
    something = something is something else THING

You could just put:
def func(something=THING):

Reply:
Right!